### ISSUE TITLE
Sound delay in games with BT 

### ISSUE THREAD

**Issue by porkmanager** [fxsound2/fxsound-app#192]
Any 3d game has sound delay when im using fx sound with bt headphones. Probably that isnt only in game but in windows it is not noticeable.
Without bluetooth headphones its works properly, no any delays.
And BT headphones connected directly(not thru sound fx) havent sound delays.

https://streamable.com/hxfm09

**Comment by doolhoofd** [comment]
Could you clarify what you mean with "3D game?"
Does this happen with only one device, or have you tested with multiple devices?
Do you have any other audio enhancement application running when this happens?
And have you read through the standard troubleshooting list? 

Some tips included are:
running FxSound as Administrator, 
disabling unnecessary audio devices, 
disabling Exclusive Mode, Enhancements, and Hardware Acceleration for all devices,
setting the problematic device as the Default Communication device, 
altering the frequency settings of your devices, 
scanning with LatencyMon, 
among others, see:
https://forum.fxsound.com/t/troubleshooting/3105

This is the main GitHub issue on Bluetooth: 
https://github.com/fxsound2/fxsound-app/issues/29

The numerous possible combinations of components involved in any digital audio setup, like varying hardware and software elements, different input and output devices, elaborate settings menus, drivers, operating system versions, and any unknown and unexpected conflicts or incompatibilities that might arise between them, inevitably creates a problem of inherent complexity. 
As a result, tracing issues down to their root causes, and finding a permanent solution, is often a challenging process.
We are aware of the fact that using FxSound together with Bluetooth connections creates various issues for many users, and we are committed to improving FxSound's Bluetooth functionalities as one of our top priorities.

**Comment by porkmanager** [comment]
> Could you clarify what you mean with "3D game?"
Nah, that isnt related to games, its just bluetooth headphones problem. Ive tested it not in the game and can see the huge delay. Without fx sound it is has not any noticeable sound lags.
https://github.com/user-attachments/assets/678a282d-46ee-4a23-96f8-d30e674e53f7

I was tried to turn off other sound sources while using bt, disabled Exclusive Mode, Enhancements, and Hardware Acceleration for all devices - nothing helps with sound delay.
Few minutes in the latency mon also showed me nothing wrong wit system latency.
![image](https://github.com/user-attachments/assets/564719db-2576-45a5-b380-c75e20ed856c)

rest of sound devices on my laptop working fine with fx sound without any noticable sound delays (internal speakers, 3.5jack, hdmi). 
I have few more bluetooth headphones i will try them latter with fx sound

**Comment by porkmanager** [comment]
Oh, On the video in the my post below the delays are not so noticeable. Have no idea how to record it properly.

**Comment by porkmanager** [comment]
Seems i found the way to record it properly. Choosen the bt headphones as a sound source in the OBS
https://github.com/user-attachments/assets/2d0837f8-5254-4a20-95ce-e4a08aed1e11
